src/zino/snmp.py

```diff
@@ -388,6 +388,18 @@ def _var_bind_is_in_scope(var: PySNMPVarBind) -> bool:
                 break  # Nothing left to query
         return dict(results)
 
+    def _is_in_scope(self, entry: MibObject, oid: Tuple[str, str]):
```

Comment:
You are calling this method from another class, which means it should probably not marked as private.  I suggest removing the leading underscore, and adding a docstring that explains its purpose/usage.